client/lib/plans/constants.js

```diff
@@ -139,18 +145,26 @@ export const plansList = {
 			'Everything included with Premium, as well as live chat support,' +
 			' unlimited access to premium themes, and Google Analytics.'
 		),
-		getFeatures: () => [ // pay attention to ordering, it is used on /plan page
-			FEATURE_CUSTOM_DOMAIN,
-			FEATURE_EMAIL_LIVE_CHAT_SUPPORT,
-			FEATURE_UNLIMITED_PREMIUM_THEMES,
-			FEATURE_ADVANCED_DESIGN,
-			FEATURE_UNLIMITED_STORAGE,
-			FEATURE_NO_ADS,
-			FEATURE_WORDADS_INSTANT,
-			FEATURE_VIDEO_UPLOADS,
-			FEATURE_GOOGLE_ANALYTICS,
-			FEATURE_NO_BRANDING
-		],
+		getFeatures: () => {
+			let features = [ // pay attention to ordering, it is used on /plan page
+				FEATURE_CUSTOM_DOMAIN,
+				FEATURE_EMAIL_LIVE_CHAT_SUPPORT,
+				FEATURE_UNLIMITED_PREMIUM_THEMES,
+				FEATURE_ADVANCED_DESIGN,
+				FEATURE_UNLIMITED_STORAGE,
+				FEATURE_NO_ADS,
+				FEATURE_WORDADS_INSTANT,
+				FEATURE_VIDEO_UPLOADS,
+				FEATURE_GOOGLE_ANALYTICS,
+				FEATURE_NO_BRANDING
+			];
+
+			if ( isEnabled( 'manage/advanced-seo' ) ) {
+				features.splice( 8, 0, FEATURE_ADVANCED_SEO );
+			}
```

Comment:
we could also use something like `lodash.compact` to generate this list without splitting it into different statements…

```js
getFeatures: () => compact( [
	FEATURE_CUSTOM_DOMAIN,
	FEATURE_EMAIL_LIVE_CHAT_SUPPORT,
	…,
	isEnabled( 'manage/advanced-seo' )
		? FEATURE_ADVANCED_SEO
		: null,
	FEATURE_GOOGLE_ANALYTICS,
	FEATURE_NO_BRANDING
] );
```